### DISCUSSION TITLE
Line number should not be selected when we want to copy.

### DISCUSSION THREAD

**Discussion by graczhual** [pradyunsg/furo#307] [Potential Issue] [answered]
In theme `sphinx_rtd_theme` and `pydata_sphinx_theme`, line number could not be selected, but in `furo`:
<img width="1423" alt="image" src="https://user-images.githubusercontent.com/87636820/142127444-c56b553f-d9d7-415e-92ce-2d9431a07bd3.png">
<img width="1224" alt="image" src="https://user-images.githubusercontent.com/87636820/142127495-45bc83bd-b136-4772-a04d-dfdd9fc64461.png">

**Comment by pradyunsg** [comment]
Got a live website for me to try this on?

**Reply by graczhual** [reply]
https://tensorbay-python-sdk--1061.org.readthedocs.build/en/1061/quick_start/examples/DogsVsCats.html#dogs-vs-cats

**Comment by pradyunsg** [comment, marked as answer]
Fixed in https://github.com/pradyunsg/furo/commit/fe6217c18dda5d79eda0a17705ee6a03850262dc.

**Reply by graczhual** [reply]
https://tensorbay-python-sdk--1061.org.readthedocs.build/en/1061/quick_start/examples/DogsVsCats.html#dogs-vs-cats

**Reply by pradyunsg** [reply]
You've built against Furo 2021.10.09, which obviously doesn't contain this unreleased change. :)

**Reply by graczhual** [reply]
So when will this change release? I have built it with Furo 2021.11.16.
https://tensorbay-python-sdk--1061.org.readthedocs.build/en/1061/quick_start/examples/BSTLD.html

**Comment by graczhual** [comment]
So when will this change release? I have built it with Furo 2021.11.16.
https://tensorbay-python-sdk--1061.org.readthedocs.build/en/1061/quick_start/examples/BSTLD.html

**Reply by pradyunsg** [reply]
When I find some time to do it. 🤷🏻‍♂️
